hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -840,6 +840,47 @@
     }
 
     // RSTMGR tests:
+    {
+      name: chip_rstrmgr_reset_info
+      desc: '''Verify the `reset_info` CSR register.
+
+            Generate all 6 types of reset, and check the reset_info is as expected.
+            '''
+      milestone: V2
+      tests: []
+    }
+    {
+      name: chip_rstrmgr_cpu_info
+      desc: '''Verify the expected values from the `cpu_info` CSR on reset.
+
+            For some software induced resets we can predict the expected contents of `cpu_info`;
+            reads of writes to unmapped addresses for example. Generate these resets and verify
+            the `cpu_info` register contents when reset is handled.
+            '''
```

Comment:
Same comment as before (AND for the below one as well) - the tests developed for other IPs that generate this scenario should be cross-referenced here. As such, no new tests need to be developed for this testpoint explicitly. This is more from the perspective of covering all of our basis and getting our documentation solid.

Reply:
I checked one of them, the rv_dm IP generates one of these resets. The testpoints don't yet refer to a specific test, so we cannot cross-reference them yet. The added TODO should be adequate.